### ISSUE TITLE
i can't find the files on runpod

### ISSUE THREAD

**Issue by kasparovabi** [williamyang1991/Rerender_A_Video#101]
i can't find the files on runpod, i started runpod with your dock but only shows a111 or comfyui files

**Comment by williamyang1991** [comment]
The runpod version is not implemented by us.
You can ask for help in this [issue](https://github.com/williamyang1991/Rerender_A_Video/issues/99).

**Comment by FurkanGozukara** [comment]
i have runpod auto installer working perfect

if it gets broken i update immediately



53.) RunPod - Cloud - Paid

[**Turn Videos Into Animation / 3D Just 1 Click - ReRender A Video Tutorial -  Installer For RunPod**](https://youtu.be/cVf9Qf_pKks)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/6MFsLs-f_GZR2L1Q6z413.png)](https://youtu.be/cVf9Qf_pKks)

**Comment by kasparovabi** [comment]
> The runpod version is not implemented by us. You can ask for help in this [issue](https://github.com/williamyang1991/Rerender_A_Video/issues/99).

yeah sorry i wrote on the wrong tab, thx for your response btw <3

**Comment by kasparovabi** [comment]
> i have runpod auto installer working perfect
> 
> if it gets broken i update immediately
> 
> 53.) RunPod - Cloud - Paid
> 
> [**Turn Videos Into Animation / 3D Just 1 Click - ReRender A Video Tutorial - Installer For RunPod**](https://youtu.be/cVf9Qf_pKks)
> 
> [![image](https://camo.githubusercontent.com/e93377070172b9d49bc77a175c8c8adf957c3525384fd8e0f911ff54023c9533/68747470733a2f2f63646e2d75706c6f6164732e68756767696e67666163652e636f2f70726f64756374696f6e2f75706c6f6164732f3633343562643839666531333464666437613064626134302f364d46734c732d665f475a52324c3151367a3431332e706e67)](https://youtu.be/cVf9Qf_pKks)

Furkan bey sizin kaynaklarınızı da inceledim ve yayıncının kendi versiyonuyla ilerlemeye çalışıyorum çünkü runpod'un masrafları biraz fazlalaştı o yüzden çok istesem de sizin patreonununzdan alamıyorum dosyanızı, yine de ilginiz için teşekkürler
